### PR TITLE
📖 docs update jobs.md for 1.26

### PR DESCRIPTION
**Pull request by aniruddha2000**
Signed-off-by: Aniruddha Basak <codewithaniruddha@gmail.com>

part of #7671 

cc @kubernetes-sigs/cluster-api-release-team

Q: Do we have to add a `1-26-latest` after line 126?

### CODE REVIEW

**Comment by sbueringer** [comment]
/hold

@aniruddha2000  Let me review the test-infra PR (https://github.com/kubernetes/test-infra/pull/28309) first (~ first January week), then we can update this PR accordingly

File: docs/book/src/reference/jobs.md [hunk 1]
```diff
@@ -73,18 +73,18 @@ Prow Periodics:
   * GINKGO_FOCUS: `[K8s-Upgrade]`
 * [periodic-cluster-api-e2e-workload-upgrade-1-24-1-25-main] `./scripts/ci-e2e.sh` FROM: `stable-1.24` TO: `stable-1.25`
   * GINKGO_FOCUS: `[K8s-Upgrade]`
-* [periodic-cluster-api-e2e-workload-upgrade-1-25-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/latest-1.26`
+* [periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/stable-1.26`
```
Comment (fabriziopandini) [inline docs/book/src/reference/jobs.md:76]: I think we are missing [periodic-cluster-api-e2e-workload-upgrade-1-26-latest-main]

File: docs/book/src/reference/jobs.md [hunk 2]
```diff
@@ -123,6 +123,6 @@ Prow Periodics:
 [periodic-cluster-api-e2e-workload-upgrade-1-22-1-23-main]: https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi-e2e-main-1-22-1-23
 [periodic-cluster-api-e2e-workload-upgrade-1-23-1-24-main]: https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi-e2e-main-1-23-1-24
 [periodic-cluster-api-e2e-workload-upgrade-1-24-1-25-main]: https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi-e2e-main-1-24-1-25
-[periodic-cluster-api-e2e-workload-upgrade-1-25-latest-main]: https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi-e2e-main-1-25-latest
+[periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main]: https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi-e2e-main-1-25-1-26
```
Comment (fabriziopandini) [inline docs/book/src/reference/jobs.md:126]: Also here, we are missing 1-26-latest

File: docs/book/src/reference/jobs.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Prow Presubmits:
   * GINKGO_FOCUS: `[IPv6] [PR-Informing]`, IP_FAMILY: `IPv6`
 * ✳️️ ✴️ [pull-cluster-api-e2e-full-main] `./scripts/ci-e2e.sh`
   * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade] [IPv6]` (i.e. "no tags")
-* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-25-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/latest-1.26`
+* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-25-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/stable-1.26`
```
Comment (sbueringer) [inline docs/book/src/reference/jobs.md:30]:
```suggestion
* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-26-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.26` TO: `ci/latest-1.27`
```

File: docs/book/src/reference/jobs.md (hunk 1 above)
Comment (sbueringer) [reply docs/book/src/reference/jobs.md:76]: Resolving as it has been added now

File: docs/book/src/reference/jobs.md [hunk 4]
```diff
@@ -73,18 +73,20 @@ Prow Periodics:
   * GINKGO_FOCUS: `[K8s-Upgrade]`
 * [periodic-cluster-api-e2e-workload-upgrade-1-24-1-25-main] `./scripts/ci-e2e.sh` FROM: `stable-1.24` TO: `stable-1.25`
   * GINKGO_FOCUS: `[K8s-Upgrade]`
-* [periodic-cluster-api-e2e-workload-upgrade-1-25-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/latest-1.26`
+* [periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/stable-1.26`
```
Comment (sbueringer) [inline docs/book/src/reference/jobs.md:76]:
```suggestion
* [periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `stable-1.26`
```

File: docs/book/src/reference/jobs.md [hunk 5]
```diff
@@ -73,18 +73,20 @@ Prow Periodics:
...
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* [periodic-cluster-api-e2e-workload-upgrade-1-26-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.26` TO: `ci/latest-1.26`
```
Comment (sbueringer) [inline docs/book/src/reference/jobs.md:78]:
```suggestion
* [periodic-cluster-api-e2e-workload-upgrade-1-26-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.26` TO: `ci/latest-1.27`
```

File: docs/book/src/reference/jobs.md (hunk 2 above)
Comment (sbueringer) [reply docs/book/src/reference/jobs.md:128]: Has been added

**Comment by sbueringer** [comment]
Thx!

/lgtm

Let's keep the hold until:
* https://github.com/kubernetes/test-infra/pull/28309 is merged
  * The cherry-picks for v1.3 and v1.2 of this PR are merged as well: https://github.com/kubernetes-sigs/cluster-api/pull/7799#issuecomment-1369430467

Just that we have support for v1.26 in the v1.3 and v1.2 branches before we change our jobs and that we actually have the jobs before we update our documentation

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Comment by sbueringer** [comment]
/cherry-pick release-1.3

(no further cherry-picks needed)

**Pull request merged**